include/multipass/utils.h

```diff
@@ -97,6 +98,18 @@ void validate_server_address(const std::string& value);
 bool valid_hostname(const std::string& name_string);
 std::string generate_mac_address();
 bool valid_mac_address(const std::string& mac);
+template <typename NetworkContainer>
+std::optional<NetworkInterfaceInfo> find_bridge_with(const NetworkContainer& networks,
+                                                     const std::string& member_network,
```

Comment:
The `member_network` seems a little weird. What this function does is to find any networks who has a link to the target network, so maybe `target_network`. Not sure whether it is a more expressive name. Besides this, since it is a `std::string`, so maybe it can be sufixed with `_id` or `_name` to emphasize the difference to `networks` name where network means a `NetworkInterfaceInfo` object.

Reply:
Yeah, `target_network` would be better. I'll update it

Not so sure about id/name. There is a balance between how much information a variable name provides and how large it is. Code looks a lot better with shorter names, especially in an already verbose language like C++. In the context of this function, the fact that the type is a string already tells you that this is a designator of some sort.

Reply:
I've updated it to `target_network`. How does that sound?